src/domain/template/template/template.service.ts

```diff
@@ -232,6 +232,7 @@ export class TemplateService {
     const template = await this.getTemplateOrFail(templateInput.id, {
       relations: {
         profile: true,
+        whiteboard: true,
```

Comment:
I suggest instead of passing true, pass `template.type === TemplateType.WHITEBOARD`and instead of loading the full entity, load only the `id` since it's the only used field.

Reply:
good point, addressed